api/src/main/java/com/flipkart/flux/api/StateMachineDefinition.java

```diff
@@ -44,19 +44,25 @@
     /* User supplied string for easy identification of a workflow instance */
     private String correlationId;
 
+    /* Client Eb Id to which task is supposed to be forwarded for execution in Flux Runtime  */
+    private String clientElbId;
```

Comment:
Shouldn't this and correlationId field be a part of hascode, toString and Equals methods ?